CHANGELOG.rst

```diff
@@ -2,15 +2,26 @@
 Changelog
 =========
 
+1.2.0 (unreleased)
+==================
+
+* Removed support for Django 1.8, 1.9, 1.10
```

Comment:
If we made a note here about added support for new Django versions, that would be great

Reply:
@vThaian I do not know if this PR already adds Django 2.x support by itself. Won't we need to test that first?